cmd/podman/stats.go

```diff
@@ -77,7 +77,9 @@ func statsCmd(c *cli.Context) error {
 
 	if ctr > 1 {
 		return errors.Errorf("--all, --latest and containers cannot be used together")
-	}
+	} else if ctr == 0 {
+		return errors.Errorf("--all, --latest, or a container must be specified")
+    }
```

Comment:
Indentation here seems a little wonky - spaces instead of a tab?

Reply:
forgot to fmt